com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/MLAPIProfilerCounterValue.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Profiling;
+using Unity.Profiling.LowLevel;
+using Unity.Profiling.LowLevel.Unsafe;
+using UnityEngine;
+
+namespace ProfilerModuleOverride
```

Comment:
We should nest this under MLAPI. For futureproofing I would also recommend that we just call the namespace Profiler since the fact that we're overriding a module is an implementation detail

Reply:
This has been nested under MLAPI now.